code/modules/mob/living/living.dm

```diff
@@ -1442,6 +1442,9 @@
 	if(!istype(L))
 		return
 
+	if(issilicon(L) || issilicon(src) || isanimal(L) || isanimal(src))
+		return // can't spread fire to mobs that don't catch on fire
```

Comment:
can't silicons catch on fire, though?

Reply:
Borgs can catch fire but it’s only for aesthetics.  They take no fire damage.  If I remember correctly, it only happens during extreme heat for the SM.